manifest.json

```diff
@@ -36,9 +39,5 @@
       "http://api.openweathermap.org/"
    ],
    "update_url": "https://clients2.google.com/service/update2/crx",
-   "version": "3.4",
-   "web_accessible_resources": [
-      "img/clock.png",
-      "sound/bells.ogg"
-   ]
```

Comment:
What was this`web_accessible_resources` array?

Reply:
If I understand https://developer.chrome.com/extensions/manifest/web_accessible_resources correctly it defined extension resources that could.be accessed from outside right?

Reply:
No clue. Hasn't been updated in a while, and doesn't seem to actually effect anything so I removed it.

Reply:
I believe so, however, this extension doesn't do anything like that so it isn't required.

Reply:
Yeah good call to remove this. Strange that this was here in the first place 😅